src/components/TableRowLink/TableRowLink.tsx

```diff
@@ -0,0 +1,50 @@
+import { TableRow, TableRowTypeMap } from "@material-ui/core";
+import { makeStyles } from "@saleor/macaw-ui";
+import clsx from "classnames";
+import React from "react";
+
+import Link from "../Link";
+
+type MaterialTableRowPropsType = TableRowTypeMap["props"];
+
+export interface TableRowLinkProps
+  extends Omit<MaterialTableRowPropsType, "onClick"> {
+  children?: React.ReactNode;
```

Comment:
Does it make sense to have this component without any children? I'd make it required